test-utils/runtime/client/src/lib.rs

```diff
@@ -148,7 +150,7 @@ impl substrate_test_client::GenesisInit for GenesisParameters {
 				storage.top.clone().into_iter().chain(child_roots).collect(),
 				sp_runtime::StateVersion::V1,
 			);
-		let block: runtime::Block = client::genesis::construct_genesis_block(state_root);
+		let block: runtime::Block = construct_genesis_block(state_root, StateVersion::V0);
```

Comment:
```suggestion
		let block: runtime::Block = construct_genesis_block(state_root, StateVersion::V1);
```

Above it uses `V1` everywhere.